File: src/main/java/com/group11/bookstore/frames/HelloFrame.java
package com.group11.bookstore.frames;

import org.springframework.stereotype.Component;

import javax.swing.*;

@Component
public class HelloFrame extends JFrame {

    public HelloFrame() {

        super("Hello, world!");

        this.setSize(200,200);
        this.setDefaultCloseOperation(EXIT_ON_CLOSE);

        this.add(new JLabel("Hello, world!", SwingConstants.CENTER));
    }
}

File: src/main/kotlin/com/group11/bookstore/frames/HelloFrame.kt
package com.group11.bookstore.frames

import org.springframework.stereotype.Component
import java.awt.Dimension
import javax.swing.JFrame
import javax.swing.JLabel
import javax.swing.SwingConstants

@Component
class HelloFrame : JFrame("Hello, world!") {

    init {

        this.size = Dimension(200, 200)
        this.defaultCloseOperation = EXIT_ON_CLOSE

        this.add(JLabel("Hello, world!", SwingConstants.CENTER))
    }
}